lib/api/src/js/store.rs

```diff
@@ -266,23 +254,23 @@ mod objects {
 }
 
     impl_store_object! {
-        functions => VMFunction,
-        tables => VMTable,
+        // Note: we store the globals in order to be able to access them later via
+        // `StoreObjects::iter_globals`.
         globals => VMGlobal,
```

Comment:
but VMGlobal are not handled as a StoreObject anymore. Isn't there a risk of unsynchronisation here?

Reply:
> Isn't there a risk of unsynchronisation here?

No. The only risk is the fact that the globals are stored in the `Store` as a side effect. But they are stored and work nonetheless.
But I think we should probably remove that after merging WASIX